packages/design-system/src/components/InlineEditing/Primitives/InlineEditingPrimitive.tsx

```diff
@@ -84,18 +121,27 @@ const InlineEditingPrimitive = forwardRef(
 			}
 		}, [hasError]);
 
+		useEffect(() => {
```

Comment:
I think the useEffect is no longer useful with this implem, as the internalValue is not used at all when controlled 🤔

Reply:
Removed!